docs/samples/kafka/kfserving_mnist.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
The service never begins available as the init container fails with CrashLoopBackoff. I assume I have not finished the setup correctly? Maybe need to add some instructions if the notebook is not standalone?

Reply:
removed the notebook, it is a bit tricky to get this working standalone since it depends on Minio, Kafka.